src/themes/bankrate/theme.json

```diff
@@ -336,6 +337,18 @@
           "backgroundColor": "veryLightGrey"
         }
       },
+      "default": {
+        "color": "grey-90"
+      },
+      "error": {
+        "color": "tomato"
+      },
+      "focus": {
+        "color": "grey-90"
+      },
+      "formInput": {
+        "boxShadow": "inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)"
```

Comment:
I would move `inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)` to be in the tokens at the top (under `shadows`) and then use the token name here instead

Reply:
I've updated and moved to `shadows`